medusa/providers/torrent/json/t411.py

```diff
@@ -73,10 +89,90 @@ def __init__(self):
         # Cache
         self.cache = tv.Cache(self, min_time=10)  # Only poll T411 every 10 minutes max
 
+    def _get_season_search_strings(self, episode):
```

Comment:
These two `_get_X_search_strings` methods share a lot of common logic.  They should be made more DRY to make the code more maintainable.